### PR TITLE
feat(events): emit affix and minimise events

### PR DESCRIPTION
**Pull request by jackhkmatthews**
boolean events emitted for affix and minimise on scroll or resize

### CODE REVIEW

File: src/app/scroll-collapse/scroll-collapse.directive.ts [hunk 1]
```diff
@@ -211,6 +223,7 @@ export class ScrollCollapseDirective implements AfterViewInit, OnDestroy {
   public calculateMinimiseMode(viewport: Viewport): void {
     this.minimiseMode =
       viewport.scrollY >= this.originalHeight + this.originalTop;
+    this.minimiseChange.emit(this.minimiseMode);
```
Comment (edoparearyee) [inline src/app/scroll-collapse/scroll-collapse.directive.ts]: should this only emit if there is a change?

File: src/app/scroll-collapse/scroll-collapse.directive.ts [hunk 2]
```diff
@@ -220,6 +233,7 @@ export class ScrollCollapseDirective implements AfterViewInit, OnDestroy {
    */
   public calculateAffixMode(viewport: Viewport): void {
     this.affixMode = viewport.scrollY >= this.originalTop - this.yOffset;
+    this.affixChange.emit(this.affixMode);
```
Comment (edoparearyee) [inline src/app/scroll-collapse/scroll-collapse.directive.ts]: should this only emit if there is a change?

**Review by edoparearyee — CHANGES_REQUESTED** [review]

File: src/app/scroll-collapse/scroll-collapse.directive.ts (hunk 1 above)
Comment (jackhkmatthews) [reply src/app/scroll-collapse/scroll-collapse.directive.ts]: Updated

File: src/app/scroll-collapse/scroll-collapse.directive.ts (hunk 2 above)
Comment (jackhkmatthews) [reply src/app/scroll-collapse/scroll-collapse.directive.ts]: Updated

**Review by edoparearyee — APPROVED** [review]

**Pull request merged**